backend/server/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java

```diff
@@ -1876,4 +1789,28 @@ public void deleteLogVersionedInputs(Session session, List<Long> repoIds) {
     query.executeUpdate();
     LOGGER.debug("ExperimentRun versioning deleted successfully");
   }
+
+  public InternalFuture<Empty> updateExperimentRunName(UpdateExperimentRunName request) {
+    final var runId = request.getId();
+    final var name = request.getName();
+    final var now = Calendar.getInstance().getTimeInMillis();
+
+    return checkPermission(
+            Collections.singletonList(runId), ModelDBActionEnum.ModelDBServiceActions.UPDATE)
+        .thenCompose(
+            unused ->
+                jdbi.withHandle(
+                    handle -> {
+                      String updateQueryString =
+                          "UPDATE experiment_run SET name=:name, date_updated=:date_updated WHERE id=:runId";
```

Comment:
maybe need to update the version, as well?  Not sure...

Reply:
I've added, I saw that other method are doing it. ✅